packages/react-server-cli/src/commands/init.js

```diff
@@ -0,0 +1,71 @@
+import _ from "lodash";
+import fs from "fs";
+import {spawnSync} from "child_process";
+import chalk from "chalk";
+import fileExists from "../fileExists";
+import ConfigurationError from "../ConfigurationError";
+
+const DEPENDENCIES = [
+	"react-server",
+	"babel-preset-react-server",
+
+	// TODO: Modernize our peer deps and remove versions here.
+	"superagent@1.2.0",
+	"react@~0.14.2",
+	"react-dom@~0.14.2",
+]
+
+const DEV_DEPENDENCIES = [
+
+	// TODO: These, too.
+	"webpack-dev-server@~1.13.0",
+	"webpack@^1.13.1",
+]
+
+const CONFIG = {
+	"routes.json": {
+		middleware: [],
```

Comment:
Maybe?  My thought was if you want a page you can generate it with `react-server add-page`.  If you don't then this way you won't have to go in and delete something you didn't ask for.